src/smc-project/configuration.ts

```diff
@@ -79,7 +79,7 @@ async function sage_info(): Promise<{
     // We need the version of sage (--version runs quickly)
     try {
       const info = (
-        await exec(`env PATH="${PATH}" sage --version`)
+        await exec("sage --version", { env: { PATH } })
```

Comment:
Oops, that sets the entire environment to have only one variable PATH, and deletes everything else from the environment.  This will cause subtle bugs.  Please rewrite this to take process.env, set the PATH in that, and then set env to that.

```
> require('child_process').exec("export", {env:{PATH:'foo'}}, console.log);0;
0
> null 'export PATH=\'foo\'\nexport PWD=\'/home/cocalc/kucalc/cluster2\'\n' '' 
```